### ISSUE TITLE
Video bit depth doesn't display in the OSC with latest mpv version.

### ISSUE THREAD

**Issue by Tempest32** [Moodkiller/MPV-Made-Easy#7]
mpv-x86_64-20200426-git-640db1e

![old](https://i.postimg.cc/FzFHYZhk/image.png)

mpv-x86_64-20200621-git-dc24a43

![new](https://i.postimg.cc/jq8s8ysv/image.png)

**Comment by Moodkiller** [comment]
Thanks for the bug report, to confirm as from mpv 0.33.0, `video-params/plane-depth` has been removed: 
   - remove video-params/plane-depth property (was too vaguely defined)

There doesn't look to be a replacement property for this other than `video-params/pixelformat` which returns `yuv420p10` as an example. I tried to get just the `10` to return, but to no avail. For now I'll just leave it as is as the other info that gets spat out is also useful - just not as nice looking.
